pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/sshddisabler/templates/scripts/disable-sshd.tpl.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
+set -e
+
+# Stop sshd service if active
+if systemctl is-active --quiet sshd.service ; then
+    systemctl stop sshd.service
+fi
+
+# Disable sshd service if enabled
+if systemctl is-enabled --quiet sshd.service ; then
+    systemctl disable sshd.service
+fi
+
+# Disabling the sshd service does not terminate already established connections
+# Kill all currently established ssh connections
+pids=$(pidof sshd || true)
```

Can you please elaborate on why `|| true` is needed?

---

I used `|| true` to silence errors from `pidof`. However I just realised that `suse-chost` does not have `pidof` and would have to find a new command to terminate shh connections.